evalml/tests/model_understanding_tests/test_metrics.py

```diff
@@ -631,3 +632,22 @@ def test_jupyter_graph_check(
         graph_roc_curve(y, y_pred_proba)
         assert len(graph_valid) == 0
         import_check.assert_called_with("ipywidgets", warning=True)
+
+
+@pytest.mark.parametrize(
+    "problem_type",
+    ["binary", "multiclass", "regression", "time series regression"],
+)
+@pytest.mark.parametrize("expected_distribution", ["matching", "not matching"])
```

Comment:
think we need a test case for the baseline case where only one class is predicted

Reply:
That's what the `non_matching` case checks!